src/pages/AdminPage.tsx

```diff
@@ -0,0 +1,203 @@
+                <div className='mx-auto'>
+                  <h3>Products</h3>
+                  <h3 className='text-2xl font-bold'>2,500</h3>
+                </div>
+              </div>
+            </div>
+          </div>
+          <div className='grid w-full grid-cols-1 gap-4 mt-4 xl:grid-cols-2 2xl:grid-cols-3'>
+            <div className='items-center justify-between p-4 bg-whiteColor my-5 rounded-lg shadow-sm sm:flex dark:border-gray-700 sm:p-6 dark:bg-gray-800'>
+              <div className='w-full'>
+                <h3 className='text-base font-normal text-gray-500 dark:text-gray-400'>New products</h3>
+                <span className='text-2xl font-bold leading-none text-gray-900 sm:text-3xl dark:text-white'>2,340</span>
+                <p className='flex items-center text-base font-normal text-gray-500 dark:text-gray-400'>
+                  <span className='flex items-center mr-1.5 text-sm text-[#0E9F6E] dark:text-green-400'>
+                    <GoArrowUp className='w-4 h-4' />
+                    12.5%
+                  </span>
+                  Since last month
+                </p>
+              </div>
+              <div className='w-full' id='new-products-chart'></div>
+            </div>
+            <div className='items-center justify-between p-4 bg-whiteColor my-5 rounded-lg shadow-sm sm:flex dark:border-gray-700 sm:p-6 dark:bg-gray-800'>
+              <div className='w-full'>
+                <h3 className='text-base font-normal text-gray-500 dark:text-gray-400'>Users</h3>
+                <span className='text-2xl font-bold leading-none text-gray-900 sm:text-3xl dark:text-white'>2,340</span>
+                <p className='flex items-center text-base font-normal text-gray-500 dark:text-gray-400'>
+                  <span className='flex items-center mr-1.5 text-sm text-[#0E9F6E] dark:text-green-400'>
+                    <GoArrowUp className='w-4 h-4' />
+                    3.4%
+                  </span>
+                  Since last month
+                </p>
+              </div>
+              <div className='w-full' id='week-signups-chart'></div>
+            </div>
+          </div>
+
+          {/* Tables */}
+          <div className='grid grid-cols-1 gap-4 md:grid-cols-2'>
+            <div className='shadow-md rounded-lg bg-[#F9F9F9] p-5'>
+              <div className='flex justify-between items-center mb-4'>
+                <div className='font-bold text-lg'>Top Sellers</div>
+                <a href='#' className='text-[#1877F2]'>
+                  See more
+                </a>
+              </div>
+              <table className='w-full text-left'>
+                <thead>
+                  <tr className='bg-[#D9D9D9]'>
+                    <th className='pb-2 px-4'>Seller name</th>
+                    <th className='pb-2 px-4'>Shop Name</th>
+                    <th className='pb-2 px-4'>No. of products</th>
+                  </tr>
+                </thead>
+                <tbody>
```

Comment:
These codes below seems hard coded. What if you make component that takes property, and while integrating, you fetch data and pass them as props to the components?